.claude/skills/product-engineering/SKILL.md

```
@@ -21,6 +21,11 @@ plus the numbers/failures you need (e.g. `<cmd> 2>&1 | tee /tmp/out.log | tail -
 read-heavy investigation to a subagent (the built-in `Explore` type) that returns just the conclusion.
 Same work, fewer tokens.
 
+> **Upstream caveat.** Everything below assumes **`devantler-tech`** work (autonomous as ever). For a
+> **non-`devantler-tech`** (upstream/third-party) repo, *creating* an issue or PR first needs the
+> maintainer's approval via the **ask tool** (contract → *Merge policy → Ask the maintainer before
+> creating ANY upstream issue or PR*); *fixing* an existing `devantler` upstream PR stays autonomous.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)